server/src/ai/ai.functions.ts

```diff
@@ -22,27 +22,104 @@ const openai = new OpenAI({
   apiKey: process.env.OPENAI_API_KEY,
 });
 
-/** makeSummary. 
+/** summarize. 
  * 
  * Takes in all reviews from a course as text and 
- * creates a 50 word summary of those reviews.
+ * generates 5 tags for those reviews describing lectures, assignments, professor,
+ * skill, and resources as well as the corresponding connotation of that tag.
  * @params a string that combines all reviews from a course
- * @returns summary of reviews
+ * @returns a dictionary containing 
+ * Summary: 50 word summary of all reviews,
+ * Tags: array of nouns, adjectives, and their corresponding connotations describing
+ * in that order, ex: ["Lectures", "Entertaining", "Positive"]
  */
-async function makeSummary(text: string) {
+async function summarize(text: string) {
   const completion = await openai.chat.completions.create({
     model: "gpt-3.5-turbo",
     messages: [
-      { role: "system", content: "You are creating a 50 word summary based on the collection of course reviews provided." },
+      {
+        role: "system", content: `
+          You are given a collection of course reviews provided where each review is separated by a /. You
```

Comment:
Maybe our Chat query could be more descriptive: (ex. ... a collection of college course reviews ...) But if we're happy with the current quality of summaries, it's probably not necessary